pkg/resources/sources/cpu.go

```diff
@@ -13,13 +14,18 @@ import (
 	core "github.com/ironcore-dev/ironcore/api/core/v1alpha1"
 )
 
-type CPU struct{}
+type CPU struct {
+	OvercommitVCPU float64
+}
 
-func NewSourceCPU() *CPU {
-	return &CPU{}
+func NewSourceCPU(options Options) *CPU {
+	return &CPU{OvercommitVCPU: options.OvercommitVCPU}
 }
 
 func (c *CPU) GetTotalResources(ctx context.Context) (core.ResourceList, error) {
+	if c.OvercommitVCPU == 0 || c.OvercommitVCPU < 0 {
```

Comment:
i recommend use https://pkg.go.dev/math#Signbit instead of < 0

Reply:
i don't think it will safe too. comparing float to specific number isn't (minimally in c language) safe operation.
so maybe something as c.OvercommitVCPU < 0.0001 can be safer.

Reply:
Using `epsilon := 1e-9` for the comparison. It will solve the issue.